molecule/resources/playbooks/prepare.yml

```diff
@@ -12,6 +12,12 @@
 
     - name: Prepare Debian hosts
       block:
+        - name: Replace buster-backports
+          ansible.builtin.replace:
+            path: /etc/apt/sources.list
+            regexp: '(^.*http://).*(\.debian\.org.*buster-backports.*$)'
+            replace: '\1archive\2'
+
```

Comment:
Could we add a conditional here to only run this against buster systems? I know the step won't do anything if the regex doesn't match, but IMHO a 'skipped' status in the ansible output would make more sense than an 'ok' for non-buster platforms.

Reply:
makes sense, do you already know how to check that?